platform/platform.go

```diff
@@ -0,0 +1,18 @@
+package platform
+
+//Platform system platform
+var Platform string
+
+const (
+	//defaultPlatform platform x86_64
+	defaultPlatform = "x86_64"
+)
+
+//GetPlatform get platform
+func GetPlatform() string {
```

Comment:
If we set default value for the `--platform` flag, we can remove this functionality entirely

Reply:
We have removed this package and we are using an default value for the flag.